msg/RobotRelative.msg

```diff
@@ -0,0 +1,18 @@
+# An another robot relative to the robot
+
+# robot without known color
+uint8 ROBOT_UNDEFINED = 1
+uint8 ROBOT_MAGENTA = 2
+uint8 ROBOT_CYAN = 3
```

Comment:
The rules (https://humanoid.robocup.org/wp-content/uploads/RC-HL-2023-Rules.pdf line 371pp) state that the colors are defined as `red` and `blue`. Please rename to the correct names.

Reply:
Done.